exercises/bank.js

```diff
@@ -1,37 +1,67 @@
 
 
 class Bank{
-	constructor(){
+	constructor(bankName){
```

Comment:
One of the most important things to consider when building an application in OOP is the functionality that each of the objects you are creating are going to need. In this particular example you are being asked to create a “bank” object, and you should notice as you look at the methods included in this Class constructor, that they are all related to the functionality a bank would need to have. For example, the object can make accounts, check to see if accounts exist, remove accounts, add money to accounts, and withdraw money from accounts. I’m sure you are thinking right now what other features could be helpful. Perhaps you would want to be able to update the name of an account, or add an interest rate… maybe even store the name of account holders? These are the kinds of things you should be considering when designing your objects, so make sure to take a moment when planning object structure so that a vital piece of functionality is not missed!